doc/api/errors.md

```diff
@@ -233,7 +233,9 @@ details of error generation from the user. For instance:
 
 ```js
 function MyError() {
-  Error.captureStackTrace(this, MyError);
+  const instance = new Error()
```

Comment:
```suggestion
  const { stackTraceLimit } = Error
  Error.stackTraceLimit = 0
  const instance = new Error()
  Error.stackTraceLimit = stackTraceLimit
```

Without this, the stack frames would be computed twice. That would be a lot of overhead that is not needed. Creating errors is actually quite expensive CPU wise.

Reply:
Good to know! I'll add a comment that explains this because otherwise I think it's a little confusing.